default.nix

```diff
@@ -1,13 +1,35 @@
 { haskellNixSrc ? builtins.fetchTarball {
-      url = "https://github.com/input-output-hk/haskell.nix/archive/a84e3b55c642e7aec57c9677158cf446d90e9759.tar.gz";
-      sha256 = "0qnchf0f8d5k6363mcrqv936wz6zljs06bjvg1i6xmkg31bdj48m";
+      url = "https://github.com/input-output-hk/haskell.nix/archive/f78841c2d05c6c686d3888131104360944464dc2.tar.gz";
+      sha256 = "1zwhskx9drd0266kcrxf5vzcgs9vfiiibzkc2h3yarfb2xl7185q";
     }
-, nixpkgs ? haskellNixSrc + "/nixpkgs"
-, pkgs ? import nixpkgs (import haskellNixSrc)
+, nixpkgs ? (import haskellNixSrc {}).sources.nixpkgs-default
+, pkgs ? import nixpkgs (import haskellNixSrc {}).nixpkgsArgs
 , haskellCompiler ? "ghc865"
```

Comment:
Should we bump this too?

Reply:
@michaelpj the compiler you mean?

Reply:
Yeah - I don't know how much it matters, though.